tools/stylelint/stylelint-color-hex-whitelist/__tests__/index.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable import/unambiguous */
```

Comment:
We should create a nested `.eslint` file to ensure we don't have to remember to write this comment on every new file.

Just like [here](https://github.com/barnardos/consent-form-builder/blob/apply-design-system/config/webpack/.eslintrc).

Reply:
👍